services/github-service-api/src/main/java/org/kontinuity/catapult/service/github/api/GitHubRepository.java

```diff
@@ -23,4 +24,18 @@
      * @return the github.com page for the repository
      */
     URI getHomepage();
+
+    /**
+	 * @param path
+	 *            the path to the file
+	 * @return the URL of the file with the given {@code} in the GitHub
+	 *         repository, <code>null</code> if no file with the given
+	 *         {@code path} exists in the repository.
+	 */
+	URI getDownloadUrl(String path);
+
+	/**
+	 * @return the URL to use to clone the project from GitHub
+	 */
+	URI getCloneUrl();
```

Comment:
And do we want to be explicit here that we're returning the HTTP transport clone URI?  Above we already have "getGitTransportUrl"; do we need both?  And if so, they should be symmetrical in naming.